person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :age, name
+  attr_reader :id
+
+  private
+
+  def of_age?
+    false unless of_age? >= 18
+  end
+
+  def can_use_services?
+    false unless of_age? && parent_permission == true
+  end
```

Comment:
- [ ] As required for the project, kindly ensure that only ` of_age?` method is `private` and `can_use_services?` method should be `public`. To fix this you can update your code to something like in the example below. Optionally you can also refactor your code to something like this:
```ruby
class Person
  ...
  
def can_use_services? 
    of_age?() || @parent_permission
  end

  private 

  def of_age?
    @age >= 18
  end
end
```

Reply:
I already see the error when using Private, thanks @tresorsawasawa  for the observation, I already made the change